modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase

```diff
@@ -2849,6 +2849,62 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11392. Can see timestamps from payment transaction history"
+	@priority = 3
+	test CanViewOrderTransactionHistoryTimestamps {
+		property portal.acceptance = "true";
```

Comment:
I see you set portal.acceptance = "true". Are we gonna add the test to the relevant?